check_run/check_run/doctype/check_run/check_run.js

```diff
@@ -399,6 +410,9 @@ function download_nacha(frm) {
 	window.setTimeout(() => {
 		frm.reload_doc()
 	}, 1000)
+	if (!frm.doc.ach_file_generated) {
+		frappe.db.set_value('Check Run', frm.doc.name, 'ach_file_generated', 1)
```

Comment:
Please refactor this to use a whitelisted function. Generally we want to avoid the use of the `frappe.db` API in JS because of the permissions issues it presents.